cics-bundle-reactor-archetype/pom.xml

```diff
@@ -48,7 +48,11 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-archetype-plugin</artifactId>
-          <version>3.1.0</version>
+          <version>3.2.1</version>
+        </plugin>
+        <plugin>
+          <artifactId>maven-site-plugin</artifactId>
```

Comment:
Is this not inherited from the parent?

Reply:
It didn't seem to, I updated the parent pom.xml first but it still asked for the other ones to be specified. Will investigate further

Reply:
I got it working by adding it the pluginManagement of the parent pom.